### PR TITLE
Library and Loadout page #2

### PR DESCRIPTION
**Pull request by erri120**
Part of #1864.

### CODE REVIEW

File: src/NexusMods.App.UI/Controls/TreeDataGrid/TreeDataGridItemModel.cs [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+                    if (isActivated)
+                    {
+                        serialDisposable.Disposable = model.ChildrenObservable
+                            .OnUI()
+                            .Bind(model._children)
+                            .SubscribeWithErrorLogging();
+                    }
+                })
+                .AddTo(disposables);
```
Comment (Al12rs) [inline src/NexusMods.App.UI/Controls/TreeDataGrid/TreeDataGridItemModel.cs:133]: This breaks when the parent item is scrolled out of view, deactivating it, while the children are still in view.

The children subscription should not be disposed when the parent is deactivated.

It should only be disposed if the parent is no longer expanded, or potentially both the parent and all the children are deactivated.

**Review by Al12rs — CHANGES_REQUESTED** [review]

**Review by halgari — APPROVED** [review]
I don't see anything crazy here, so LTGM.

**Comment by halgari** [comment]
Ran some manual tests on this, I'm not seeing a scroll bar on the new library. I assume this will be in the next PR?

**Comment by erri120** [comment]
> Ran some manual tests on this, I'm not seeing a scroll bar on the new library. I assume this will be in the next PR?

Already fixed.

**Review by Al12rs — CHANGES_REQUESTED** [review]
Tested latest commit but Library isn't showing new mod page entries if you have the Library open and download a new mod

**Comment by erri120** [comment]
> Tested latest commit but Library isn't showing new mod page entries if you have the Library open and download a new mod.

That's expected, still a WIP.

**Review by Al12rs — APPROVED** [review]

**Pull request merged**